relnotes

```diff
@@ -225,14 +225,15 @@ create_body () {
   echo "[Documentation](https://docs.k8s.io) &" \
        "[Examples](https://releases.k8s.io/$CURRENT_BRANCH/examples)"
   echo
-  if [[ -n $release_tars ]]; then
-    echo "## Downloads for $title"
+  if [[ -n ${release_tars} ]]; then
```

Comment:
this file doesn't follow the bash style guide, but please revert these changes for consistency.